### DISCUSSION TITLE
Ray-Ban Meta glasses interfering with Dexcom G7?

### DISCUSSION THREAD

**Discussion by Chromatticism** [NightscoutFoundation/xDrip#3642] [Q&A]
I have been using the G7 with xdrip for some time with no problem. About a month ago, I was on a canoe trip and was doing a portage and pulled my G7 off. I tried to set up a new sensor but I couldn't get it to pair. I thought maybe it was because I didn't put the old sensor in the microwave like I usually do. When I got back to civilization (but not home - I was staying with a friend) I tried troubleshooting, but still couldn't get it to work, so I removed that sensor, put it in my friend's microwave, and tried setting up ANOTHER new sensor, but had the same problem - it wouldn't pair. I even tried installing the Dexcom app and connecting with that, but it wouldn't connect either. Since I was frustrated with TWO sensors in a row being wasted, once I got home I purchased a Libre and set that up with xdrip, which worked fine. Then when that Libre expired I tried a G7 with xdrip again, which worked fine. When THAT G7 expired, I happened to be on another trip, this time staying with a friend. I was careful to put the old sensor far away before setting up my new sensor. Unfortunately I got the same problem, and again couldn't connect with the Dexcom app. It just wouldn't connect - there was no pairing request. 

It occurred to me that the commonality between these two problems was that I was on a vacation when I tried to start the sensors. I was using my Ray-Ban Meta glasses both times - I generally only use them when I'm on vacation (to take pictures). The glasses have some intense connectivity both through Bluetooth and Wifi to use the built-in virtual assistant and transfer photos and video, and have a persistent notification on my phone when I'm using them. Perhaps they are somehow interfering with pairing the G7, or corrupting the sensor by confusing it with their transmissions? I've also had some problems downloading data from the glasses - they keep telling me to disable my VPN, even though I don't have one.

When I got home I put the glasses in the basement, uninstalled the Dexcom app, restarted my phone, and tried pairing the G7 with xdrip again. This time it did stop saying "hunting transmitter", but I still never got a pairing request, and it started saying "Mismatch - wait" even though the code was definitely correct, and it also said there were "Scanning Errors". I removed that sensor, wrapped it in aluminum foil, put it in a different room, and started ANOTHER new sensor - and this time I got a pairing request and eventually it started working normally and I am currently getting glucose values.

I don't have any screenshots from the morning when I tried to connect without getting the glasses out of the picture - I don't think it said it connected at all then.

I had gone through all the recommended settings here https://navid200.github.io/xDrip/docs/Dexcom/G7.html and I did try setting the sensor code to something else and then back to the correct code several times. I also tried uninstalling and re-installing xdrip. None of it made any difference. 

My question is, do you think it is plausible that the Ray-Ban Meta glasses are the source of the problem? 

![WhatsApp Image 2024-08-25 at 22 38 26_e0dfe40b](https://github.com/user-attachments/assets/530bf70b-4806-4289-9bd7-cae0f05b9886)

![WhatsApp Image 2024-08-25 at 22 39 05_69a56f11](https://github.com/user-attachments/assets/465fbe9e-e1ff-48b3-b9cb-70005bdb2f7a)

![WhatsApp Image 2024-08-25 at 22 40 38_9bfda7ee](https://github.com/user-attachments/assets/1339d7a5-f9ee-4126-8888-4354ca82e864)

**Comment by Navid200** [comment]
I don't see why not.
Considering this seems to affect xDrip and Dexcom app both, I suggest we do some research online to see if we can find any other reports.
And Dexcom may have heard of it too.
I will let you know if I find anythng.

Thanks for reporting this.
